app/components/dsfr_component/base.rb

```diff
@@ -5,6 +5,8 @@ class DsfrComponent::Base < ViewComponent::Base
 
   delegate :config, to: Dsfr::Components
 
+  TITLE_TAGS = %i[h1 h2 h3 h4 h5 h6].freeze
```

Comment:
I don't think we need to enforce the header level. It should be a number, and that's about it.

Reply:
fine with me but there are indeed only 6 levels cf https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements